pinot-broker/src/main/java/com/linkedin/pinot/broker/queryquota/HitCounter.java

```diff
@@ -0,0 +1,58 @@
+/**
+ * Copyright (C) 2014-2016 LinkedIn Corp. (pinot-core@linkedin.com)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.linkedin.pinot.broker.queryquota;
+
+public class HitCounter {
```

Comment:
Can you add a description of what it is that you're doing in this class? From reading the code, it seems like you're averaging the number of queries over the last 100 time buckets.